src/pages/Failure.tsx

```diff
@@ -3,14 +3,11 @@ import React from 'react'
 import { useHistory } from 'react-router'
 
 import StatsLayout from '../components/layouts/StatsLayout'
-import { useStorage } from '../useStorage'
 import useStore from '../useStore'
 
 const Failure = () => {
   const history = useHistory()
 
-  useStorage()
```

Comment:
I removed this everywhere except on `Home`. I think it doesn't make sense to override the state from the JSON everytime we switch a page and I found it can interfere/cause rerenders. It would need to be done such that it only overrides the store if it is not initialized yet.

Reply:
Yes I think that's the best for now. Thanks for trying it out. We can try to optimize later on.